src/main/java/openfasttrack/core/Linker.java

```diff
@@ -113,35 +125,56 @@ private void linkIgnoringRevision(final LinkedSpecificationItem item,
                 .getByIdIgnoringVersion(id);
         if (coveredLinkedItems.isEmpty())
         {
-            // FIXME: Collect list of covered specification items that do not
-            // exist
-            // https://github.com/hamstercommunity/openfasttrack/issues/86
-            System.err.println("Error: " + item.getId() + " links to non-existent " + id);
+            linkOrphanToStaleId(item, id);
         }
         else
         {
-            for (final LinkedSpecificationItem itemCoveredIgnoringVersion : coveredLinkedItems)
+            linkToOutdatedOrPredated(item, id, coveredLinkedItems);
+        }
+    }
+
+    private void linkOrphanToStaleId(final LinkedSpecificationItem item,
+            final SpecificationItemId id)
+    {
+        final LinkedSpecificationItem deadItem = findOrCreateStaleItem(id);
+        item.addLinkToItemWithStatus(deadItem, LinkStatus.ORPHANED);
+    }
+
+    private LinkedSpecificationItem findOrCreateStaleItem(final SpecificationItemId id)
+    {
+        LinkedSpecificationItem staleItem = this.staleIndex.get(id);
+        if (staleItem == null)
+        {
+            staleItem = new LinkedSpecificationItem(new SpecificationItem.Builder().id(id).build());
+            this.staleIndex.put(id, staleItem);
+        }
```

Comment:
You could also use [Map.putIfAbsent()](https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#putIfAbsent-K-V-)

Reply:
In this case I always would have to build the stale Item (i.e. the value) on speculation.